tests/ci/workflow_jobs_lambda/app.py

```diff
@@ -141,8 +155,12 @@ def handler(event: dict, context: Any) -> dict:
         logging.error("The event data: %s", event)
         logging.error("The context data: %s", context)
 
-    # We record only finished steps
-    steps = len([step for step in wf_job["steps"] if step["conclusion"] is not None])
+    if killed_job(wf_job):
+        # for killed job we record 0
+        steps = 0
```

Comment:
we use `step == 0` later as a sign that job was killed. correct?
shall we add dedicated "conclusion" value for such jobs?

Reply:
I'd leave it as it is in the event. FInished failure with step=0 is already a mark. Previously it was `<5`, so now it will be very much the sign

Reply:
Now `steps` is the only thing we mess up. All others are left as is. It's because we don't write the actual structures, but rewrite it to just a number, see line 165+